locale/fi/sidenavigation.json

```diff
@@ -0,0 +1,74 @@
+{
+  "title": "Sivunavigaatio (SideNavigation)",
+  "intro": "Sivunavigaatiota käytetään sisältösivujen välillä navigointiin erityisesti silloin kun sivut on ryhmitelty useaan eri alakategoriaan.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Komponentin otsikon ja navigaatioelementtien tekstisisältöjen tulee olla kuvaavia, ja ne on hyvä pyrkiä pitämään mahdollisimman lyhyinä.",
+    "Ruudunlukijasaavutettavuuden varmistamiseksi tulee huolehtia, että yksittäisille navigaatioelementeille (linkit) asetetaan asianmukaiset aria-attribuutit: aria-current=page tällä hetkellä aktiiviselle sivulle, aria-current=location aktiivisen sivun yläsivulle, aria-label mikäli halutaan välittää linkin tekstisisällön ulkopuolista informaatiota ja aria-disabled mikäli linkki on estetty.",
+    "Komponentille tulee antaa ylätason aria-label, joka kertoo ruudunlukijalle navigaation kontekstin ja tarkoituksen.",
+    "Sivunavigaatiolle on lisäksi hyvä antaa HTML-id, jotta siihen voidaan liikkua suoraa esimerkiksi SkipLink-navigaatioelementeistä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Sivunavigaatio on navigaatioelementti, jonka sisällä on lista. Komponentin tekninen toteutus mahdollistaa sen, että lista voi sisältää tavanomaisten linkkien lisäksi esimerkiksi painikkeita tai muita semanttisia elementtejä. Kaikki listan elementit ovat kuitenkin semantiikasta huolimatta visuaaliselta tyyliltään samanlaisia. Varsinaisen linkkilistan lisäksi komponenttiin kuuluu sen otsikko sekä siihen liittyvä ikoni."
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Sivunavigaatiossa linkit ovat hierarkisesti puurakenteessa. Komponentissa on mahdollista näyttää yhdestä kolmeen tasoa linkkejä. Navigaatioelementin ollessa aktiivisena sen alasivut ovat näkyvissä."
```

Comment:
"Komponentissa on mahdollista näyttää yhdestä kolmeen tasoa linkkejä." could probably be simplified a bit in terms of language. Something like "Komponentissa on mahdollista näyttää linkkejä kolmessa tasossa"

Reply:
Check!